vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ModuleLinkManager.java

```diff
@@ -104,4 +110,14 @@ protected void deleteLinkRepo(IModuleLink link) {
         moduleLinkRepo.delete((ModuleLink) link);
     }
 
+    @Override
+    public HashSet <ISpace> findModuleLinksFromModuleId(String moduleId) {
```

Comment:
please fix your ide settings, we don't do blanks before `<`

Reply:
this method is called find**ModuleLinks**FromModuleId, why is it returning spaces?

Reply:
Fixed the ide setting and renamed the method name to findSpaceLinksFromModuleId